### ISSUE TITLE
Enable Cloud base METAR processing

### ISSUE THREAD

**Issue by JeffHamiltonNOAA** [NOAA-GSL/VxLegacyIngest#546]
After meeting with the model physics team, the decision was made to enable a cloud base verification ingest that will look at already existing METAR output and determine the cloud base to be the first instance of a cloud categorization (everything but clear or vertical visibility) and store in the database similar to what we do with ceilings. Additionally, we want to track how often the cloud ceiling (using the old algorithm) is the first cloud level indicated (essentially how often the ceiling and base are the same). Incorporate code to track this as well. The modelers will be working on the cloud base diagnostic inside UPP as well, but I will create a separate issue to address that once ready

**Comment by mollybsmith-noaa** [comment]
Need to have this ready by September 30th.

**Comment by JeffHamiltonNOAA** [comment]
I have created the new cloud base directory on Jet as a copy of the ceiling ingest. Working to modify the preprocessing scripts before running in real time

**Comment by JeffHamiltonNOAA** [comment]
Observations are being processed and stored in the new cloud_base sub-database within MySQL. I am now just waiting on model data to make further adjustments

**Comment by JeffHamiltonNOAA** [comment]
I see that I specified that this issue would just handle the observations. Therefore this issue can be closed
